### PR TITLE
Update base-fr.yaml

### PR DESCRIPTION
**Pull request by jojolebestof**
All the update is not finished. 
Also I have reorder some lines according to base-en.yaml to simplify the comparizon.

Based on base-en.yaml from the commit f584d9d93e0404bce2af1ccf3acf79aabbd0a220

### CODE REVIEW

**Comment by jojolebestof** [comment]
The CI fail is due to translations/**base-el.yaml** not translations/base-fr.yaml

File: translations/base-fr.yaml [hunk 1]
```diff
@@ -190,7 +193,9 @@ dialogs:
     confirmSavegameDelete:
         title: Confirmez la suppression
         text: >-
-            Êtes-vous sûr de vouloir supprimer votre partie ?
+            Êtes-vous sûr de vouloir supprimer cette partie ?<br><br>
+            '<savegameName>' au niveau <savegameLevel><br><br>
+            Cela sera définitif!
```
Comment (LeopoldTal) [inline translations/base-fr.yaml:198]: Add ` ` (U+202F NARROW NO-BREAK SPACE) before `!`

File: translations/base-fr.yaml [hunk 2]
```diff
@@ -259,20 +269,19 @@ dialogs:
...
+        desc: Donnez-lui un nom, vous pouvez aussi inclure <strong>le raccourci</strong> d’une forme (que vous pouvez générer <a href="https://viewer.shapez.io" target="_blank">ici</a>).
 
     markerDemoLimit:
         desc: Vous ne pouvez créer que deux balises dans la démo. Achetez la version complète pour en placer autant que vous voulez !
 
     exportScreenshotWarning:
         title: Exporter une capture d’écran
-        desc: >-
-            Vous avez demandé à exporter une capture d’écran de votre base. Soyez conscient que cela peut s’avérer passablement lent pour une grande base, voire faire planter votre jeu !
+        desc: Vous avez demandé à exporter une capture d’écran de votre base. Soyez conscient que cela peut s’avérer passablement lent pour une grande base, voire faire planter votre jeu !
```
Comment (LeopoldTal) [inline translations/base-fr.yaml:280]: Better with block chomping IMO, avoids overlong line

File: translations/base-fr.yaml [hunk 3]
```diff
@@ -360,7 +382,10 @@ ingame:
         noShapesProduced: Aucune forme produite pour le moment.
 
         # Displays the shapes per minute, e.g. '523 / m'
-        shapesPerMinute: <shapes> ⁄ m
+        shapesDisplayUnits:
+            second: <shapes> / s
```
Comment (LeopoldTal) [inline translations/base-fr.yaml:386]: Should be ` ⁄ ` (U+00A0 NO-BREAK SPACE and U+2044 FRACTION SLASH)

File: translations/base-fr.yaml [hunk 4]
```diff
@@ -402,47 +433,41 @@ ingame:
...
-        white: Blanc
-        uncolored: Sans couleur
-        black: Noir
-    shapeViewer:
-        title: Calques
-        empty: Vide
-        copyKey: Copier le raccourci de la forme
+    # Connected miners
+    connectedMiners:
+        one_miner: 1 Extracteur
```
Comment (LeopoldTal) [inline translations/base-fr.yaml:438]: Don't capitalise `Extracteur`; use no-break spaces

File: translations/base-fr.yaml [hunk 5]
```diff
@@ -506,49 +541,75 @@ buildings:
         default:
             name: &painter Peintre
             description: &painter_desc Colorie entièrement la forme de gauche avec la couleur de droite.
+
+        mirrored:
+            name: *painter
+            description: Colorie entièrement la forme de gauche avec la couleur de gauche.
```
Comment (LeopoldTal) [inline translations/base-fr.yaml:547]: That doesn't look right. Both inputs can't be on the left, can they?

File: translations/base-fr.yaml [hunk 6]
```diff
@@ -506,49 +541,75 @@ buildings:
...
-        deliver: Livrez
-        toUnlock: pour débloquer
-        levelShortcut: NV
+
     wire:
         default:
             name: Câble
-            description: Permet de transporter de l’énergie.
-    advanced_processor:
+            description: &wire_desc Transfère des signaux, qui peuvent être des éléments, des couleurs ou des booléens (1 / 0). Les câbles de couleurs différentes ne se connectent pas.
```
Comment (LeopoldTal) [inline translations/base-fr.yaml:570]: "éléments" doesn't work here. Maybe "objets"?

File: translations/base-fr.yaml [hunk 7]
```diff
@@ -506,49 +541,75 @@ buildings:
...
+            name: &wire_tunnel Tunnel à câble
+            description: Permet de croiser deux câbles sans les connecter entre eux.
+
+    constant_signal:
         default:
-            name: Générateur d’énergie
-            description: Génère de l’énergie en consommant des formes.
-    wire_crossings:
+            name: &constant_signal Constante
+            description: Émets un signal constant, qui peut être une forme, une couleur ou un booléen (1 / 0).
```
Comment (LeopoldTal) [inline translations/base-fr.yaml:584]: typo: `Émets` → `Émet` (fix throughout)

File: translations/base-fr.yaml [hunk 8]
```diff
@@ -506,49 +541,75 @@ buildings:
...
+    logic_gate:
+        default:
+            name: Porte ET
+            description: Émets un booléen "1" si les deux entrées sont vraies.
+        not:
+            name: Porte INVERSEUR
+            description: Émets un booléen "1" si l'entrée n'est pas vraie.
+        xor:
+            name: Porte OU-EXCLUSIF
+            description: Émets un booléen "1" si une des entrée est vraie, mais pas les deux.
```
Comment (LeopoldTal) [inline translations/base-fr.yaml:600]: typo: `entrée` → `entrées` (next component too)

File: translations/base-fr.yaml [hunk 9]
```diff
@@ -506,49 +541,75 @@ buildings:
...
-        merger:
-            name: Fusionneur de câble
-            description: Fusionne deux câbles en un seul.
+            name: &lever Interrupteur
+            description: Peut être basculé pour émettre un signal booléen (1 / 0) dans le calque de câblage, qui peut être utilisé pour contrôler un filtre de forme par exemple.
+
+    logic_gate:
+        default:
+            name: Porte ET
+            description: Émets un booléen "1" si les deux entrées sont vraies.
```
Comment (LeopoldTal) [inline translations/base-fr.yaml:594]: Could you use curly quotes (throughout)? I know they're horrid but that would be consistent with the rest of the file. (Or, like, guillemets, but aaaarghhh.)

File: translations/base-fr.yaml [hunk 10]
```diff
@@ -506,49 +541,75 @@ buildings:
...
+            name: Porte OU-EXCLUSIF
+            description: Émets un booléen "1" si une des entrée est vraie, mais pas les deux.
+        or:
+            name: Porte OU
+            description: Émets un booléen "1" si une des entrée est vraie.
+
+    transistor:
+        default:
+            name: &transistor Transistor
+            description: &transistor_desc Transmet l'entrée inférieure si l'entrée latérale est vraie (une forme, couleur ou "1").
```
Comment (LeopoldTal) [inline translations/base-fr.yaml:608]: Phrasing's kinda awkward but I don't really have a better idea. `inférieure` → `du bas` would be an improvement at least.

File: translations/base-fr.yaml [hunk 11]
```diff
@@ -587,7 +648,9 @@ storyRewards:
 
     reward_miner_chainable:
         title: Extracteur en série
-        desc: Vous avez débloqué <strong>l’extracteur en série</strong> ! Il permet de <strong>transférer ses ressources</strong> à d’autres extracteurs pour augmenter le débit sortant !
+        desc: >-
+            Vous avez débloqué <strong>l’extracteur en série</strong> ! Il permet de <strong>transférer ses ressources</strong> à d’autres extracteurs pour augmenter le débit sortant !<br><br>
+            Note : L'ancien extracteur a été remplacé dans votre barre d'outils maintenant !
```
Comment (LeopoldTal) [inline translations/base-fr.yaml:653]: For consistency with the rest of the file please use ` ` U+00A0 NO-BREAK SPACE before `:`, ` ` U+202F NARROW NO-BREAK SPACE before `!`, and `’` U+2019 RIGHT SINGLE QUOTATION MARK. (Pauvres mouches…)

File: translations/base-fr.yaml [hunk 12]
```diff
@@ -794,6 +869,21 @@ settings:
             description: >-
                 Le jeu est divisé en secteurs de 16×16 cases. Si ce réglage est activé, les limites de chaque secteur sont affichées.
 
+        pickMinerOnPatch:
+            title: Choisir un extracteur sur un emplacement de ressource.
+            description: >-
+                Activé par défaut, sélectionne l'extracteur si vous utilisez la pipette lors du survol d'un emplacement de ressource.
+
+        simplifiedBelts:
+            title: Convoyeurs simplifiés (Laid)
```
Comment (LeopoldTal) [inline translations/base-fr.yaml:878]: `moche` is totally a better translation than `laid`

File: translations/base-fr.yaml [hunk 13]
```diff
@@ -794,6 +869,21 @@ settings:
             description: >-
                 Le jeu est divisé en secteurs de 16×16 cases. Si ce réglage est activé, les limites de chaque secteur sont affichées.
 
+        pickMinerOnPatch:
+            title: Choisir un extracteur sur un emplacement de ressource.
```
Comment (LeopoldTal) [inline translations/base-fr.yaml:873]: "emplacement de ressource" is weird. "mine de ressources", maybe?

File: translations/base-fr.yaml [hunk 14]
```diff
@@ -794,6 +869,21 @@ settings:
...
+            description: >-
+                Activé par défaut, sélectionne l'extracteur si vous utilisez la pipette lors du survol d'un emplacement de ressource.
+
+        simplifiedBelts:
+            title: Convoyeurs simplifiés (Laid)
+            description: >-
+                Désactive le rendu des éléments des convoyeurs sauf lors du survol du convoyeurs pour économiser des performances. Je ne recommande pas de jouer avec ce réglage si vous n'avez pas absolument besoin de performance.
```
Comment (LeopoldTal) [inline translations/base-fr.yaml:880]: typo: `du convoyeurs`. Singular and plural both sound fine to me, but not both at once

File: translations/base-fr.yaml [hunk 15]
```diff
@@ -838,52 +934,55 @@ keybindings:
...
-        advanced_processor: Inverseur de couleur
-        energy_generator: Générateur d’énergie
-        wire: Câble
 
 about:
     title: À propos de ce jeu
     body: >-
-        Ce jeu est open source et développé par <a href="https://github.com/tobspr"
-        target="_blank">Tobias Springer</a> (c’est moi).<br><br>
+        Ce jeu est open source et développé par <a href="https://github.com/tobspr" target="_blank">Tobias Springer</a> (c’est moi).<br><br>
```
Comment (LeopoldTal) [inline translations/base-fr.yaml:977]: Why did you un-fold lines? It's way more readable with shorter lines.

File: translations/base-fr.yaml [hunk 16]
```diff
@@ -140,23 +143,23 @@ demoBanners:
...
         Désolé, ce jeu sera lent sur votre navigateur web ! Procurez-vous la version complète ou téléchargez Chrome pour une meilleure expérience.
 
     savegameLevel: Niveau <x>
     savegameLevelUnknown: Niveau inconnu
-
-    continue: Continuer
-    newGame: Nouvelle partie
-    madeBy: Créé par <author-link>
-    subreddit: Reddit
+    savegameUnnamed: Sans nom
```
Comment (LeopoldTal) [inline translations/base-fr.yaml:162]: `Sans titre` is more idiomatic IMO

**Review by LeopoldTal — COMMENTED** [review]
Not sure if it's wise to do this now before tobspr officially syncs the translation keys.

Looks good, I mostly have typography quibbles.

File: translations/base-fr.yaml (hunk 12 above)
Comment (jojolebestof) [reply translations/base-fr.yaml]: ok!

File: translations/base-fr.yaml (hunk 14 above)
Comment (jojolebestof) [reply translations/base-fr.yaml]: Oops!

File: translations/base-fr.yaml (hunk 15 above)
Comment (jojolebestof) [reply translations/base-fr.yaml:977]: This is to simplify the file comparizon with the english version.
But yes it is true, it is more readable with shorter lines.
Maybe in the english version also the line should be cutted.

File: translations/base-fr.yaml (hunk 16 above)
Comment (jojolebestof) [reply translations/base-fr.yaml]: Yep

File: translations/base-fr.yaml (hunk 2 above)
Comment (jojolebestof) [reply translations/base-fr.yaml:280]: It is for file comparizon with english file.

File: translations/base-fr.yaml (hunk 1 above)
Comment (jojolebestof) [reply translations/base-fr.yaml]: I agree with you for no-break-space but I have modified the file inside the web browser. I do not know how to have this char.
Have you an idea?

File: translations/base-fr.yaml (hunk 3 above)
Comment (jojolebestof) [reply translations/base-fr.yaml:386]: I have tried to copy but I am not shure it is good.

File: translations/base-fr.yaml (hunk 4 above)
Comment (jojolebestof) [reply translations/base-fr.yaml]: Capitalise OK
No-break-space TODO

File: translations/base-fr.yaml (hunk 5 above)
Comment (jojolebestof) [reply translations/base-fr.yaml]: You right. I hope my correction is better than this one

**Comment by LeopoldTal** [comment]
I've opened [jojolebestof#1](https://github.com/jojolebestof/shapez.io/pull/1) for pedantry.

**Comment by tobspr** [comment]
Awesome, thanks!

**Pull request merged**